x-pack/plugins/security_solution/public/common/components/sourcerer/helpers.tsx

```diff
@@ -142,3 +145,15 @@ export const getPatternListWithoutSignals = (
   patternList: string[],
   signalIndexName: string | null
 ): string[] => patternList.filter((p) => p !== signalIndexName);
+
+export const getSourcererDataview = async (
```

Comment:
lets put this in `x-pack/plugins/security_solution/public/common/containers/sourcerer/api.ts`